### PR TITLE
Add empty line after checkboxes and before warning

### LINKED ISSUE #276: Incorrect alignment of the warning
When you have created a wiki without specifying the patch or a wiki is created with the merged patch(es), there is a warning.

There isn't an empty line before warning and after checkbox for "Previously generated wikis".

See:
image

I have a fix for this:
image

### PR DESCRIPTION
**Pull request by kizule**
Fixes #276.

### CODE REVIEW

File: css/common.css [hunk 1]
```diff
@@ -98,6 +98,10 @@ summary > .oo-ui-labelElement-label {
 
 /* Wiki list */
 
+.oo-ui-messageWidget.oo-ui-messageWidget-block.oo-ui-flaggedElement-warning {
+	margin-top: 16px;
```
Comment (edg2s) [inline css/common.css:102]: use 1em for the margin

Comment (edg2s) [inline css/common.css:102]: Use 1em for the margin

File: css/common.css [hunk 2]
```diff
@@ -98,6 +98,10 @@ summary > .oo-ui-labelElement-label {
 
 /* Wiki list */
 
+.oo-ui-messageWidget.oo-ui-messageWidget-block.oo-ui-flaggedElement-warning {
```
Comment (edg2s) [inline css/common.css:101]: Add a new class to the widget using `'classes' => [ 'closedWikiWarning' ]`, then just use that class as the selector here: `.closedWikiWarning {`

**Review by edg2s — CHANGES_REQUESTED** [review]

File: css/common.css (hunk 1 above)
Comment (kizule) [reply css/common.css]: I found that .showClosed had -6px for margin-top.
In index.php I found that this element is for the warning box. I've removed it from common.css

File: css/common.css (hunk 2 above)
Comment (kizule) [reply css/common.css]: See the previous comment.

File: css/common.css [hunk 3]
```diff
@@ -100,7 +100,6 @@ summary > .oo-ui-labelElement-label {
 
 .showClosed {
 	float: right;
-	margin-top: -6px;
```
Comment (edg2s) [inline css/common.css:103]: This negative margin is for the alignment of the button within the warning box, not the box itself.

**Review by edg2s — CHANGES_REQUESTED** [review]

**Comment by MatmaRex** [comment]
It doesn't work, I think `.showClosed` is the wrong element.

**Comment by kizule** [comment]
I don't know how to do this, so someone else should take a look in this.

**Comment by edg2s** [comment]
See #283

**Pull request closed without merging**